core/iwasm/compilation/aot.h

```diff
@@ -81,23 +82,9 @@ typedef enum AOTFloatCond {
 typedef struct AOTImportMemory {
     char *module_name;
     char *memory_name;
-    uint32 memory_flags;
-    uint32 num_bytes_per_page;
-    uint32 mem_init_page_count;
-    uint32 mem_max_page_count;
+    AOTMemory memory;
```

Comment:
How about adding `typedef WASMMemoryType AOTMemoryType` in the above, and here: `AOTMemoryType mem_type`?

Reply:
Agreed.